src/main/java/org/carlmontrobotics/commands/RampMaxRPMDriving.java

```diff
@@ -0,0 +1,45 @@
+package org.carlmontrobotics.commands;
+
+import static org.carlmontrobotics.Constants.Effectorc.*;
+
+import org.carlmontrobotics.subsystems.Arm;
+import org.carlmontrobotics.subsystems.IntakeShooter;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+
+// TODO: where do we use this command?
+public class RampMaxRPMDriving extends Command {
+  private final IntakeShooter intakeouttake;
+
+  public RampMaxRPMDriving(IntakeShooter intake) {
+    addRequirements(this.intakeouttake = intake);
+  }
+
+  @Override
+  public void initialize() {
+  // outtake ramps up once the 2 distance sensors detect a note
+  if(intakeouttake.intakeDetectsNote() && intakeouttake.outtakeDetectsNote()){
+        intakeouttake.setMaxOuttake(1);
```

Comment:
forgot to tell you this, but we want to ramp the outtake to 4000 rpm. You can just use the `setRPMOuttake()` from `IntakeShooter.java` to do this.